src/main/java/uk/ac/ebi/eva/contigalias/service/AssemblyService.java

```diff
@@ -69,11 +70,11 @@ public Optional<AssemblyEntity> getAssemblyOrFetchByAccession(String accession)
         if (entities.isPresent()) {
             enaDataSource.addENASequenceNamesToAssembly(entities);
```

Comment:
I think this should also be moved elsewhere, as it potentially fetches data from ENA - it should happen on PUT but not GET, I guess?

Reply:
Actually looks like it already happens in [fetchAndInsert](https://github.com/nitin-ebi/contig-alias/blob/91dabfc01ebfbf38ee381f01518f539003bb94ff/src/main/java/uk/ac/ebi/eva/contigalias/service/AssemblyService.java#L116) so maybe it's okay to just remove it here.

Reply:
Method deleted